stage1/init/init.go

```diff
@@ -48,6 +49,16 @@ package main
 //   return sd_pid_get_slice(pid, slice);
 // }
 //
+// long
+// getTotalMemory()
```

Comment:
please be concise with existing C function naming s/getTotalMemoery/get_total_memory/ 
and add quick docstring (I even don't now is it returns b/kb/mb)